srcpkgs/ibus-chewing/template

```diff
@@ -3,28 +3,22 @@ pkgname=ibus-chewing
 version=1.5.1
 revision=1
 archs="x86_64"
-build_style=gnu-makefile
-hostmakedepends="git cmake pkg-config"
-makedepends="gettext-devel libchewing-devel ibus-devel gob2 libX11-devel gtk+3-devel glib-devel findutils"
+build_style=cmake
+configure_args="-DLIBEXEC_DIR=/usr/libexec"
+make_cmd=make
+hostmakedepends="git cmake pkg-config findutils gettext-devel"
+makedepends="libchewing-devel ibus-devel gob2 libX11-devel gtk+-devel glib-devel"
 depends="ibus libchewing"
 short_desc="Chewing input method for ibus"
 maintainer="ctoid <funk443@yandex.com>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/definite/ibus-chewing"
-
-do_fetch() {
-	git clone https://github.com/definite/ibus-chewing.git $wrksrc
-}
+distfiles="https://github.com/definite/ibus-chewing/archive/refs/tags/${version}.tar.gz"
+checksum="4c0b58e9b13b826969d8df6276275bbd893a4a6d665cc218a65aa3eecde00d53"
+nocross=yes
 
 pre_configure() {
 	rm -rf cmake-fedora
 	git clone https://pagure.io/cmake-fedora.git
```

Comment:
why is this needed?

Reply:
You mean the cmake-fedora part?

Reply:
Yes, if the project actually depends on a specific git submodule, we generally add new distfile (tar.gz for a specific commit) to it and move it into the right place.

Reply:
I added this part because the original cmake-fedora host is no longer available.